src/scss/components/hero-banner/_hero-banner.scss

```diff
@@ -3,15 +3,48 @@
 // This Hero banner is only for the front page of site on VPUE subtheme
 
 .front {
+  
+  // Remove the red brand bar
+  .su-brand-bar {
+    display: none;
+  }
+
+  // Moving the site's global message alert to the top when
+  .ds-entity--stanford-global-message {
+    .su-alert {
+      position: relative;
+      z-index: 15;
+    }
+  }
+
+  //skip nav look
+  .su-skipnav {
+    z-index: 16;
+    background-color: $su-color-white;
+  }
+
+  //search
+  .su-masthead {
+    .su-site-search__input {
+      background: transparent;
+      border: 1px solid $su-color-white;
+      color: $su-color-white;
+      font-family: $su-font-sans;
+
+      &::placeholder {
+        color: $su-color-white;
+      }
+    }
+  }
 
   // Move the image up more to compensate for the admin interface.
   &.role--authenticated {
 
-    //drupal messages to the top 
+  //drupal messages to the top 
     .page-content {
       > .su-alert {
         position: relative;
-        top: -286px;
+        top: -290px;
         z-index: 15;
       }
     }
```

Comment:
Is this code block needed still? It seemed to work just fine when I commented it out, due to the other rule already applied

Reply:
updated.